src/artificial_contrast/data.py

```diff
@@ -1,26 +1,38 @@
 import os
-from pathlib import Path
+
 import pandas as pd
-from fastai.vision import get_transforms, SegmentationItemList
+
+from fastai.vision import get_transforms, Path, SegmentationItemList
+from artificial_contrast.settings import TEST_SET_EXAMINATIONS
 
 
 CODES = ['no_contrast', 'contrast']
 
 
-def get_scans(data_path):
+def get_scans(data_path, test=False):
     scans = []
 
-    patients = sorted(os.listdir(data_path))
-
+    patients = get_patients(data_path, test)
     for patient in patients:
         bc_dir = Path(data_path / patient / 'BC')
         scans += [str(p) for p in bc_dir.ls()]
 
     return sorted(scans)
 
 
-def get_patients(data_path):
-    return sorted(os.listdir(data_path))
+def get_patients(data_path, test=False):
```

Comment:
IMO (as written later) `get_patients` shouldn't change with logic below, but extract test patients to `test_data_path`.
Asserts prolly belong to `scripts` but placing it here has a nice feature of "ensuring assert early". Maybe that expected counts should be just and optional keyword to pass